### DISCUSSION TITLE
How do I create an organized bank?

### DISCUSSION THREAD

**Discussion by jmfreeland** [christofmuc/KnobKraft-orm#257] [General]
Hi,

I've been using the MKS-80 functionality to test out patches, but I'd like to create an organized bank of only my favorites. I don't seem to be able to rearrange anything or copy/paste. Is there a way to create a new bank from my collection of sounds or organize them generally?

Cheers,
J

**Comment by christofmuc** [comment]
Yes, there is, if you use at least version 2.0.

2.0.5 is the latest release, i think.

In the tree on the left, you can expand the MKS-80 synth, and create a user bank. Give it an arbitrary name, and for the MKS-80 which has only one bank there is not much to choose as its preset to Bank A. Then, on the right hand side a grid is shown into which you can drag and drop sounds from the main library. The bank is empty in the beginning, so the first patch will fill the whole list (think Init patch), but of course you can drag more patches into it. Click save to database to persist the user bank. I haven't tested sending a bank into the MKS-80, not sure if this would work given there are tones and patches in there, maybe needs more work.

Alternatively, you can always create a list in the tree and just drag patches into a list. I use these to collect patches that go well together, but list are across synths so you can store patches from different synths in there e.g. for a song.

**Reply by jmfreeland** [reply]
Ok great. I was trying to drag to the grid in the middle, so I think I get this workflow and can give it a try. I don't believe the 80 accepts bank sends, so I may just create a bank and use it locally. Thank you for the help!

**Comment by christofmuc** [comment]
Closing as answered.
